test/types.js

```diff
@@ -204,4 +204,13 @@ describe('Types', function () {
       [new NSObject init];
     `).should.match(/#"<NSObject: 0x[0-9a-f]+> \(of doom\)"/);
   });
+
+  it('should support "choose()"', function() {
+    cycript.execute(`
+      var a = [[NSColor alloc] init];
+      var b = [[NSColor alloc] init];
```

Comment:
In case the parser supports `const` and `let` – which I can't remember if it does – we should avoid `var` and use modern syntax instead.

Reply:
I used ```const``` since it won't be changed.